core/src/it/scala/org/ensime/core/JavaCompilerSpec.scala

```diff
@@ -390,7 +390,8 @@ class JavaCompilerSpec extends EnsimeSpec with OptionValues
       }
     }
   }
-  it should "support Java 8 JavaFx addListener features" in {
+
+  it should "support Java 8 JavaFx addListener lambda features" in {
```

Comment:
The JavaFx addListener but is probably not relevant, I suspect this is a more general lambda thing.